test/lib/tracing_service_test.rb

```diff
@@ -0,0 +1,14 @@
+require 'test_helper'
+require 'minitest/autorun'
+require 'mocha/mini_test'
```

Comment:
this is impacted by #1249 , so just need to keep an eye on this depending on what's merged first